packages/react/src/ImageSlider/ImageSlider.tsx

```diff
@@ -5,40 +5,57 @@
 
 import { ChevronLeftIcon, ChevronRightIcon } from '@amsterdam/design-system-react-icons'
 import clsx from 'clsx'
-import { Children, forwardRef, useEffect, useRef, useState } from 'react'
-import type { ForwardedRef, HTMLAttributes, KeyboardEventHandler, PropsWithChildren } from 'react'
+import { forwardRef, KeyboardEvent as ReactKeyboardEvent, useEffect, useRef, useState } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
 import { ImageSliderContext } from './ImageSliderContext'
 import { ImageSliderItem } from './ImageSliderItem'
 import { ImageSliderScroller } from './ImageSliderScroller'
 import { ImageSliderThumbnails } from './ImageSliderThumbnails'
+import { Ratio } from '../AspectRatio'
 import { IconButton } from '../IconButton'
+import { Image } from '../Image/Image'
+
+export type SlideProps = {
+  src: string
+  /** Prove a URL to the image */
+  ratio: Ratio
+  /** Define an aspect ratio to use on all images */
+  alt: string
+  /** Describe to image */
+  srcSet?: string
+  /** Provide a src-set array to use responsive images */
+  sizes?: string
+  /** Provide a sizes attribute to each image */
+}
 
 export type ImageSliderProps = {
+  /** An array of images to display */
+  slides: SlideProps[]
   /** Show navigation controls */
   controls?: boolean
   /** Show native scrollbar inside gallery */
   scrollbar?: boolean
-  /** Prevent passing over possible snap elements */
-  snapstop?: boolean
   /** Show thumbnails */
   thumbnails?: boolean
   /** Label for the previous button */
   previousLabel?: string
   /** Label for the next button */
   nextLabel?: string
-} & PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+  /** Label for the image if you need to translate the alt text */
+  imageLabel?: string
+} & HTMLAttributes<HTMLDivElement>
 
 export const ImageSliderRoot = forwardRef(
   (
     {
-      children,
       className,
+      slides,
```

Comment:
Please alphabetically sort all lists like this, such as fields in types, restructuring assignments, function parameters, etc.

Reply:
Done.